docs/INSTALLATION.md

```diff
@@ -0,0 +1,263 @@
+```
+
+Once the services have started, you can check their status by running:
+
+```sh
+docker ps
+```
+
+The `ps` command will list containers along with their `CONTAINER_ID`. Then, to check logs of an individual container, run:
+
+```sh
+docker logs <CONTAINER_ID>
+```
+
+To stop the Docker runtime run:
+
+```sh
+docker-compose -f docker-compose.ove.yml down
+```
+
+To clean-up the Docker runtime first stop any active instances and then run:
+
+```sh
+docker system prune
+docker volume prune
+```
+
+## Installation by downloading and compiling source code
+
+All OVE projects use a build system based on [Lerna](https://lernajs.io/). [**OVE Asset Services**](https://github.com/ove/ove-asset-services) are based on [.NET Core](https://github.com/dotnet/core). All other OVE projects are based on [Node.js](https://nodejs.org/en/), compiled with [Babel](http://babeljs.io/), and deployed on a [PM2](http://pm2.keymetrics.io/) runtime.
+
+### Prerequisites
+
+* [git](https://git-scm.com/downloads)
+* [Node.js](https://nodejs.org/en/) (v8.0+)
+* [NPM](https://www.npmjs.com/)
+* [NPX](https://www.npmjs.com/package/npx) (install with the command: `npm install -global npx`)
+* [PM2](http://pm2.keymetrics.io/) (install with the command: `npm install -global pm2`)
+* [Lerna](https://lernajs.io/) (install with the command: `npm install -global lerna`)
+
+Compiling source code for the Docker runtime also requires:
+
+* [Docker](https://www.docker.com/get-started)
+* [Docker Compose](https://docs.docker.com/compose/install/)
+
+Building [**OVE Asset Services**](https://github.com/ove/ove-asset-services) also requires:
+
+* [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)
+* [NetVips](https://github.com/kleisauke/net-vips)
+* [MariaDB](https://mariadb.org/)
+
+The [SVG App](https://github.com/ove/ove-apps/tree/master/packages/ove-app-svg) requires:
+
+* [Tuoris](https://github.com/fvictor/tuoris) (installation instructions available on GitHub repository)
+
+### Downloading source code
+
+All OVE projects can be downloaded from their GitHub repositories:
+
+* OVE Core: [master](https://github.com/ove/ove) | [releases](https://github.com/ove/ove/releases)
+* OVE Apps: [master](https://github.com/ove/ove-apps) | [releases](https://github.com/ove/ove-apps/releases)
+* OVE Services: [master](https://github.com/ove/ove-services) | [releases](https://github.com/ove/ove-services/releases)
+* OVE Asset Services: [master](https://github.com/ove/ove-asset-services) | [releases](https://github.com/ove/ove-asset-services/releases)
+
+The `master` branch of each repository contains the latest code, and can also be cloned if you intend to contribute code or fix issues:
+
+```sh
+git clone https://github.com/ove/ove
+```
+
+Once the source code has been downloaded OVE can be installed either onto a PM2 runtime or a Docker runtime. The two approaches are explained below.
+
+### Compiling source code for the PM2 runtime using Lerna
```

Comment:
```suggestion
### Compiling source code using Lerna without Docker
```

"for the PM2 runtime" might cause confusion for novice users, who might think that this step compiles the code of pm2 itself.

Reply:
I don't expect this to be a concern. Line above reads: `Once the source code has been downloaded OVE can be installed either onto a PM2 runtime or a Docker runtime` and this follows. We have a similar title for Docker below. We also don't say `of the PM2 runtime`. I'm not taking in this suggestion, therefore.

Reply:
Another reason that I dislike this title is that PM2 is actually used as the process manager whether or not it is run inside Docker.

Reply:
That point is valid, but the user is either running their code locally on a PM2 runtime, or locally on a Docker runtime, which has been explained clearly in the docs. I think we can be explicit about it without being vague. You don't run anything on Lerna, you build it. Likewise, you will be running something on PM2, so it is worth telling it. They also have a choice of using a separate runtime locally, however, if they are using our Dockerfiles, they don't have a choice of changing what runs inside Docker (actually that is also not their concern). So, I see nothing wrong in this description.

Reply:
I'm resolving this conversation based on the explanation above.